exporting/tests/test_exporting_engine.c

```diff
@@ -678,6 +680,103 @@ static void test_simple_connector_worker(void **state)
     simple_connector_worker(instance);
 }
 
+static void test_sanitize_json_string(void **state)
+{
+    (void)state;
+
+    char *src = "check \t\\\" string";
+    char dst[19 + 1];
+
+    sanitize_json_string(dst, src, 19);
+
+    assert_string_equal(dst, "check _\\\\\\\" string");
+}
+
+static void test_sanitize_graphite_label_value(void **state)
+{
+    (void)state;
+
+    char *src = "check ;~ string";
+    char dst[15 + 1];
+
+    sanitize_graphite_label_value(dst, src, 15);
+
+    assert_string_equal(dst, "check____string");
+}
+
+static void test_sanitize_opentsdb_label_value(void **state)
+{
+    (void)state;
+
+    char *src = "check \t\\\" #&$? -_./ string";
+    char dst[26 + 1];
```

Comment:
The string `*src` is 29 bytes long

Reply:
do you really want to copy  less bytes from src?

Reply:
After sanitizing it becomes a 26-character string.

Reply:
It's a test. I'm not copying, I'm checking the sanitizing function. I don't need more bytes in `dst`.

Reply:
All right, I only wanted to be sure about what we are testing here, thanks!